bin/z-wave-graph.py

```diff
@@ -31,7 +31,9 @@ def __init__(self, config):
 
         if base_url != 'localhost':
             if 'api_password' in self.haconf['http']:
-                api_password = self.haconf['http']['api_password']
+                api_password = str(self.haconf['http']['api_password'])
```

Comment:
Re-reviewing this and I'm wondering if the check for int is required? You've just set it to a string...